test/isochrone.cc

```diff
@@ -94,16 +103,27 @@ void test_isochrones() {
 // Test auto isochrone with one contour
 // 32bit builds fail in release mode we'll look at this separately
 #if _WIN64 || __amd64__
-  std::string expected1 = "\"type\":\"LineString\"";
-  const auto test_request1 =
-      R"({"locations":[{"lat":52.078937,"lon":5.115321}],"costing":"auto","contours":[{"time":15}],"polygons":false,"denoise":0.2,"generalize":150})";
-  try_isochrone(reader, loki_worker, thor_worker, test_request1, expected1);
-
-  // Try pedestrian isochrone with one contour, polygon=true
-  std::string expected2 = "\"type\":\"Polygon\"";
-  const auto test_request2 =
-      R"({"locations":[{"lat":52.078937,"lon":5.115321}],"costing":"bicycle","contours":[{"time":15}],"polygons":true,"denoise":0.2})";
-  try_isochrone(reader, loki_worker, thor_worker, test_request2, expected2);
+  {
```

Comment:
Just curious, why are these within a scope?

Reply:
The smaller scopes, the less likely you have unintended side effects. It visually conveys pretty strongly too that there are two test-cases without having to do a bunch of plumbing with new functions etc.

Reply:
Or are these multiple steps of the same test?